src/library_eventloop.js

```diff
@@ -90,14 +90,14 @@ LibraryJSEventLoop = {
     emClearImmediate(id);
   },
 
-  emscripten_set_immediate_loop__sig: 'vpp' ,
+  emscripten_set_immediate_loop__sig: 'ipp' ,
```

Comment:
The function is declared void, why change the return type?

Reply:
Oh, the PR changes that, I see...

Reply:
Actually you are right.. the C declaration was void.  So I can just remove the return statement here..